etc/profile-a-l/localsend_app.profile

```diff
@@ -0,0 +1,64 @@
+# Firejail profile for localsend_app
+# Description: An open source cross-platform alternative to AirDrop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include localsend_app.local
+# Persistent global definitions
+include globals.local
+
+blacklist /usr/libexec
+
+noblacklist ${HOME}/.local/share/localsend_app
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.local/share/localsend_app
+whitelist ${HOME}/.local/share/localsend_app
+whitelist /usr/share/localsend_app
+include whitelist-common.inc
```

Comment:
```suggestion
#include disable-xdg.inc

#mkdir ${HOME}/.local/share/localsend_app
#whitelist ${HOME}/.local/share/localsend_app
#whitelist ${DOWNLOADS}
#whitelist /usr/share/localsend_app
#include whitelist-common.inc
```

The PR enables the profile by default on firecfg and the program appears to be
intended to be used on arbitrary user files.